docs/developer/testing/AttatchingIDEDebuggers.md

```diff
@@ -0,0 +1,66 @@
+
+# Attatching IDE Debuggers to project.
+
+## VSCode
+Create `.vscode/launch.json` file with following content
+
+```json
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Debug E2E Test",
+            "type": "go",
+            "request": "launch",
+            "mode": "test",
+            "program": "${workspaceRoot}/tests/e2e/e2e_suite_test.go",
+            "env": {
+                // "KUBECONFIG": "/path/to/.kube/config",
+                "KUBERNETES_MASTER": "http://localhost:8080",
```

Comment:
localhost:8080 4reelz?

Reply:
It's the default. Not defining it is also fine.

Reply:
Your kubeconfig takes care of what cluster it talks to.